packages/inventory-compliance/src/ComplianceEmptyState.js

```diff
@@ -11,16 +12,48 @@ import {
     EmptyStateIcon
 } from '@patternfly/react-core';
 import { CloudSecurityIcon } from '@patternfly/react-icons';
+import gql from 'graphql-tag';
+import { useQuery } from '@apollo/react-hooks';
+import { Spinner } from '@redhat-cloud-services/frontend-components';
+import { ErrorCard } from './PresentationalComponents';
+const COMPLIANCE_API_ROOT = '/api/compliance';
 
-const ComplianceEmptyState = ({ title, mainButton }) => (
-    <Bullseye>
+const QUERY = gql`
+{
+    profiles(search: "external = false") {
+        totalCount
+    }
+}
+`;
+
+const ComplianceEmptyState = ({ title, mainButton, client }) => {
+    let { data, error, loading } = useQuery(QUERY, { fetchPolicy: 'network-only', client: client });
```

Comment:
You could also just pass down the count of policies from `SystemQuery` and avoid another query, can you not?

Reply:
Unfortunately not - in SystemQuery we get the list of *policies associated to a host*, but in here I want the list of *all internal policies*.

Also, this component is used from `compliance-frontend`, in the CompliancePolicies component I could do that, but in the Reports component, we're only fetching the list of policies with `has_test_results = true`. So I'd have to change 2 out of 3 components to make an additional query to fetch the count of internal policies always when I only care if I want to load the empty state.